File: PM_Backendfra/src/main/java/dk/sep3/passwordmanager/client/viewmodel/PasswordManager.java
package dk.sep3.passwordmanager.client.viewmodel;

import dk.sep3.passwordmanager.client.service.PasswordService;

public class PasswordManager {
  private final PasswordService passwordService;

  public PasswordManager(PasswordService passwordService) {
    this.passwordService = passwordService;
  }

  public void savePassword(String username, String password) {
    passwordService.savePassword(username, password);
  }

  public void getPassword(String username) {
    String password = passwordService.getPassword(username);
    if (password != null) {
      System.out.println("Password for " + username + ": " + password);
    } else {
      System.out.println("No password found for " + username);
    }
  }
}

File: PM_Frontend/Client/src/main/C#/Core/PasswordManager.cs
using PM_Frontend.C#.Services;

namespace PM_Frontend.C#.Core
{
    public class PasswordManager
    {
        public PasswordManager(PasswordService passwordService)
        {
            passwordService = passwordService;
        }

        public void SavePassword(string username, string password)
        {
            passwordService.SavePassword(username, password);
        }

        public void GetPassword(string username)
        {
            var password = passwordService.GetPassword(username);
            if (password != null)
            {
                System.Console.WriteLine($"Password for {username}: {password}");
            }
            else
            //Simple check, can be changed when we do more with secuirty
            {
                System.Console.WriteLine($"No password found for {username}");
            }
        }
    }
}